kanto_kuska/edge_client.py

```diff
@@ -0,0 +1,150 @@
+import time
+import json
+import paho.mqtt.client as mqtt

```

Comment:
Include paho.mqtt in requirements.txt.
https://www.freecodecamp.org/news/python-requirementstxt-explained/